docker/README.md

```diff
@@ -0,0 +1,4 @@
+# Dockerfile for GDRNPP
+Configure volume-paths in docker_run.sh before usage  
```

Comment:
where is docker_run.sh?

Reply:
sorry, that line is deprecated since a docker-compose file is used now, I updated the branch